### PR TITLE
add .npmignore file

### PR DESCRIPTION
**Pull request by osher**
refers to #136

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,13 @@
   "bin": {
     "pino": "./pretty.js"
   },
+  "files" : [
+    "pino.js", 
+    "browser.js",
+    "pretty.js",
+    "LICENEE",
```
Comment (mcollina) [inline package.json]: this should be LICENSE

Comment (osher) [inline package.json]: that's against the standard
besides - tests are 40K and that doubles the size of the package.
FYI.

Comment (osher) [inline package.json]: I adhered.
let me know if you change your mind again, I'll be happy to change it.

Comment (osher) [inline package.json]: oups. almost missed. I had a typo and it still included LISENSE?
So we don't need to mention it. npm granbs it anyway.

**Comment by osher** [comment]
Here's what we got so far:
image

**Comment by mcollina** [comment]
this brings down the size of tgz from 460K to 28K, pretty awesome. Thanks!

**Comment by osher** [comment]
peasure working with you.
You're more responsive than I am!

**Comment by davidmarkclements** [comment]
Thanks @osher!

**Pull request merged**